catch-uncomitted.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+const { execFileSync } = require('child_process')
+const path = require('path')
+
+const argv = process.argv.slice(2)
+
+try {
+  execFileSync(`${path.join(__dirname, 'catch-uncomitted.sh')}`, argv, {
```

Comment:
I think that you don't actually need the extra string template here, since `path.join` already returns a string, right?
```suggestion
  execFileSync(path.join(__dirname, 'catch-uncomitted.sh'), argv, {
```

Reply:
There is also a typo in the script name: `uncomitted` with a single `m` instead of `uncommitted` with double `m` -- which suggests that this PR has not been tested...

Reply:
True! I think as you identify below the problem is that the test isn't actually using the new script. The script was only tested as working within our monorepo, where the spelling mistake originated from.

Reply:
Good shout!